2-mandatory/1-writers.js

```diff
@@ -61,6 +61,17 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
+for (let item of writers){
+
+let firstName = item[0];
+let lastName = item[1];
+let age = item[3];
+let occupation =item[2];
+}
```

Comment:
Remember to indent the inside of loops, ifs and functions. Will Make it much easier for you to follow the logic of your own code